dev-workflow-ui/webContent/resources/css/workflow-ui.css

```diff
@@ -266,11 +266,11 @@ body .custom-accordion.ui-accordion .ui-accordion-header:first-child {
   padding-right:0px;
   margin-bottom:0.15em;
 }
-.notification-scroll {
-  overflow:scroll;
-  max-height:600px;
+.ui-datascroller-content.ui-widget-content.ui-corner-all {
+  border: 0px;
+  padding: 2px;
 }
-.notification-scroll::-webkit-scrollbar {
+.ui-datascroller-content.ui-widget-content.ui-corner-all::-webkit-scrollbar {
```

Comment:
you shouldnt overwrite default primefaces css!
you should keep `notification-scroll` class
or create a different class